src/middlewared/middlewared/apidocs/templates/websocket/jobs.md

```diff
@@ -0,0 +1,123 @@
+## Jobs
+
+Tasks which require time to execute are tagged as jobs.
```

Comment:
Tasks which require significant time to execute or process significant volume of input or output are tagged as jobs.